### PR TITLE
LPS-198569 Default Value options are not shown side by side in the model builder

### PR DESCRIPTION
**Pull request by larissacribeiro**
**Related Issue**: [LPS-198569](https://liferay.atlassian.net/browse/LPS-198569)

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @cgmonte ,
I'm assigning this PR to you because it is frontend. Feel comfortable assigning to anybody from your squad :+1:

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarRoot.tsx [hunk 1]
```diff
@@ -43,6 +43,15 @@ export function RightSideBarRoot({children}: IRightSidebarRoot) {
 			return;
 		}
 
+		if (
```
[the remark this answers is not in the document]
Comment (larissacribeiro) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarRoot.tsx]: Hey @cgmonte, sorry about that, I've just updated it! Thanks

**Comment by larissacribeiro** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarRoot.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (cgmonte) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarRoot.tsx]: No problem, really :pray:

**Review by cgmonte — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:forward:force

**Pull request closed without merging**